framework/src/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -430,15 +430,29 @@ public boolean hasAnnotation(Class<? extends Annotation> a) {
     }
 
     /**
+     * Returns the actual effective annotation mirror used to annotate this type,
+     * whose Class equals the passed annoClass if one exists, null otherwise.
+     *
+     * @param annoClass annotation class
+     * @return the annotation mirror for anno
+     */
```

Comment:
Perhaps maybe add:
`@see getEffectiveAnnotations` for more information on effective annotations